### ISSUE TITLE
Error when saving xcube-sh

### ISSUE THREAD

**Issue by maximlamare** [dcs4cop/xcube-sh#48]
When trying to export a Sentinel-2 L2A cube created with `xcube-sh` as a netCDF with the following command

`cube.to_netcdf("path") `

 I get the following error:

```
TypeError: illegal data type for attribute b'history', must be one of dict_keys(['S1', 'i1', 'u1', 'i2', 'u2', 'i4', 'u4', 'i8', 'u8', 'f4', 'f8']), got O
```

Printing `cube.history` yields:

```
[{'program': 'xcube_sh.chunkstore.SentinelHubChunkStore',
  'cube_config': {'dataset_name': 'S2L2A',
   'band_names': ['B01',
    'B02',
    'B03',
    'B04',
    'B05',
    'B06',
    'B07',
    'B08',
    'B8A',
    'B09',
    'B11',
    'AOT',
    'SCL',
    'SNW',
    'CLD',
    'CLP',
    'CLM',
    'sunAzimuthAngles',
    'sunZenithAngles',
    'viewAzimuthMean',
    'viewZenithMean',
    'dataMask'],
   'band_units': None,
   'band_sample_types': 'FLOAT32',
   'tile_size': [1135, 882],
   'bbox': [551075.116667, 6798615.084978, 596475.116667, 6833895.084978],
   'spatial_res': 10,
   'crs': 'http://www.opengis.net/def/crs/EPSG/0/3857',
   'time_range': ['2020-07-01T00:00:00+00:00', '2020-07-31T00:00:00+00:00'],
   'time_period': '2 days 00:00:00',
   'time_tolerance': None,
   'collection_id': None,
   'four_d': False}}]
```

**Comment by forman** [comment]
This is caused by new xarray v0.17. There is a new behaviour that produces `DatetimeGregorian` instances for the `time` coords. But only in some case. Investigating...

**Comment by forman** [comment]
Should be fixed in 0.7

**Comment by maximlamare** [comment]
This is still happening in `v 0.8.0`
